cfgov/unprocessed/css/enhancements/typography.less

```diff
@@ -129,6 +129,11 @@ ul:last-child,
     margin-bottom: 0;
 }
 
+
+.eyebrow {
+    .heading-5();
+}
+
```

Comment:
Do you no longer need the media query to modify the margin between the eyebrow and its H1?

Reply:
That margin is specifically added between h5/.h5 and h1/.h1 at small screen sizes in [cf-base](https://github.com/cfpb/capital-framework/blob/canary/src/cf-core/src/cf-base.less#L198-L203), so using the mixin makes the override unnecessary.